subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/dependencies/AbstractVersionConstraint.java

```diff
@@ -93,7 +92,6 @@ public String getDisplayName() {
         }
         append("reject", rejectedVersionsString(), builder);
         append("branch", getBranch(), builder);
-        append("subgraph", isForSubgraph(), builder);
```

Comment:
There is a second _append_ method which I added just for this with the following signature:
`append(String name, boolean active, StringBuilder builder)`

That method can now be removed again.

Reply:
removed